index.js

```diff
@@ -149,8 +148,7 @@ module.exports = {
     'shorthand-property-no-redundant-values': true,
     'string-no-newline': true,
     'string-quotes': 'double',
-    'stylelint-disable-reason': 'always-before',
-    'time-no-imperceptible': true,
+    'time-min-milliseconds': 100,
```

Comment:
How about something smaller, such as `0.01`? Perhaps `0.1` is a bit too restrictive for things like animations perhaps? probs?